### PR TITLE
Release preparations

### PR DESCRIPTION
**Pull request by gunnarmorling**

### CODE REVIEW

File: sources/introduction.asciidoc [hunk 1]
```diff
@@ -13,9 +13,24 @@ It also offers method and constructor validation facilities to ensure constraint
...
-* TODO
+The following persons have actively contributed to Bean Validation 2.0 as members of the JSR 380 expert group and the community at large in alphabetical order:
+
+* Matt Benson
+* Emmanuel Bernard (Red Hat, Inc.)
+* Linda DeMichiel (Oracle)
+* Hendrik Ebbers (Canoo AG)
+* Hardy Ferentschik (Red Hat, Inc.)
+* Christian Kaltepoth (ingenit GmbH & Co. KG)
+*	Werner Keil
```
Comment (gsmet) [inline sources/introduction.asciidoc]: Is there a reason why some are prepended by a tab and the others by a space?

File: sources/whatsnew.asciidoc [hunk 2]
```diff
@@ -9,9 +9,21 @@
...
-
-TODO
+The main contribution of Bean Validation 2.0 is leveraging the new language features and API additions of Java 8 for the purposes of validation.
+The changes include:
+
+* Support for the new date/time data types for `@Past` and `@Future` (see <<builtinconstraints>>)
+fine-grained control over the current time and time zone used for validation (see <<constraintsdefinitionimplementation-validationimplementation-temporal-validators>>)
+* Advanced support for validating container elements by annotating type arguments of generic type, e.g. `List<@Min(1) Integer> positiveNumbers` (see <<appendix-value-extraction>>);
+this also includes:
+** More flexible cascaded validation of collection types, e.g. `Map<@Valid CustomerType, @Valid Customer`
```
Comment (gsmet) [inline sources/whatsnew.asciidoc]: Missing `>` in the Map definition.

File: sources/whatsnew.asciidoc [hunk 3]
```diff
@@ -9,9 +9,21 @@
...
 === What's new in 2.0
 
-The main contribution of Bean Validation 2.0 is leveraging the new language features of Java 8 for the purposes of validation.
-
-TODO
+The main contribution of Bean Validation 2.0 is leveraging the new language features and API additions of Java 8 for the purposes of validation.
+The changes include:
+
+* Support for the new date/time data types for `@Past` and `@Future` (see <<builtinconstraints>>)
+fine-grained control over the current time and time zone used for validation (see <<constraintsdefinitionimplementation-validationimplementation-temporal-validators>>)
```
Comment (gsmet) [inline sources/whatsnew.asciidoc]: Is there a punctuation missing or something between line 15 and 16? It looks weird.

**Comment by gunnarmorling** [comment]
Thanks, @gsmet. Pushed an update.

**Comment by gsmet** [comment]
Squashed the last commit in the previous one and merged! Thanks.

**Pull request closed without merging**